### PR TITLE
Restructuring the documentation

### PR DESCRIPTION
**Pull request by nikolasburk**
- [x] Structure
    - [x] ORM
    - [x] Accelerate
    - [x] Pulse
    - [x] Platform
    - [x] About
- [x] Links
    - [x] Put in place redirects
        - Check this PR for more info: https://github.com/prisma/docs/pull/5508
    - [x] Fix internal links
      - [x] Absolute links
      - [x] Absolute links with anchors
      - [x] Relative links
    - [x] Double-check moved images
    - [x] Remove instances of `<span class="concepts"></span>` and `<span class="guides"></span>` (maybe also `<span class="reference"></span>`?)
- [x] Other
    - [x] Hide **Platform** and **About** tabs
    - [x] Clean up frontmatters
    - [x] Review and polish index pages
      - https://github.com/prisma/docs/issues/5518

---

- [x] Figure our sidenav issues: https://github.com/prisma/docs/issues/5477

### CODE REVIEW

**Comment by janpio** [comment]
> maybe also <span class="reference"></span>

I think that is useful information - we still have the different areas in the documentation.

File: config.ts [hunk 1]
```diff
@@ -115,7 +115,7 @@ const siteConfig = {
       {
         title: `What is Prisma`,
         description: 'An overview of what Prisma ORM is and how it works.',
-        url: 'concepts/overview/what-is-prisma',
+        url: 'orm/prisma-client/queries/crud',
```
Comment (janpio) [inline config.ts]: Wrong link now.
The CRUD link in line 124 actually also needs to be updated (to this target)

Comment (nikolasburk) [reply config.ts]: doings the lord's work here, Jan! thanks for catching it 🙏

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/index.mdx [hunk 2]
```diff
@@ -10,15 +10,15 @@ dbSwitcher: ['mongodb']
 
 <TopBlock>
 
-Learn how to add Prisma to an existing Node.js or TypeScript project by connecting it to your database and generating a Prisma Client for database access. The following tutorial introduces you to the [Prisma CLI](/concepts/components/prisma-cli), [Prisma Client](/concepts/components/prisma-client), and [Prisma Introspection](/concepts/components/introspection).
+Learn how to add Prisma to an existing Node.js or TypeScript project by connecting it to your database and generating a Prisma Client for database access. The following tutorial introduces you to the [Prisma CLI](/orm/tools/prisma-cli), [Prisma Client](/orm/prisma-client), and [Prisma Introspection](/orm/prisma-schema/introspection).
```
Comment (ankur-arch) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/index.mdx]:
```suggestion
Learn how to add Prisma to an existing Node.js or TypeScript project by connecting it to your database and generating a Prisma Client for database access. The following tutorial introduces you to [Prisma CLI](/orm/tools/prisma-cli), [Prisma Client](/orm/prisma-client), and [Prisma Introspection](/orm/prisma-schema/introspection).
```

**Pull request merged**